rust/parquet/src/arrow/schema.rs

```diff
@@ -1389,21 +1450,21 @@ mod tests {
                 Field::new("c19", DataType::Interval(IntervalUnit::DayTime), false),
                 Field::new("c20", DataType::Interval(IntervalUnit::YearMonth), false),
                 Field::new("c21", DataType::List(Box::new(DataType::Boolean)), false),
-                Field::new(
-                    "c22",
-                    DataType::FixedSizeList(Box::new(DataType::Boolean), 5),
-                    false,
-                ),
-                Field::new(
-                    "c23",
-                    DataType::List(Box::new(DataType::List(Box::new(DataType::Struct(
-                        vec![
-                            Field::new("a", DataType::Int16, true),
-                            Field::new("b", DataType::Float64, false),
-                        ],
-                    ))))),
-                    true,
-                ),
+                // Field::new(
```

Comment:
These are still failing roundtrip, so I've disabled them. I'm planning on opening a JIRA to address this.